Amplify/Categories/DataStore/Query/QueryPaginationInput.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// A simple struct that holds pagination information that can be applied queries.
+public struct QueryPaginationInput {
+
+    /// The default page size
+    public static var defaultLimit: UInt = 100
```

Comment:
Seems like this should be a `let` not a `var` -- if customers want to change the limit, seems like we should force them through calling the `page` function, rather than allow them to update a `defaultLimit` (which can be changed throughout the life of an app)

Reply:
Good call!